package.json

```diff
@@ -20,12 +20,14 @@
     "LICENSE"
   ],
   "dependencies": {
+    "@eqworks/react-labs": "^1.9.0",
+    "@material-ui/core": "^4.11.0",
```

Comment:
can we move these two into `peerDependencies` (with a copy in `devDependencies`) so this package itself is leaner, also because it's anyway used in snoke which would have these two deps.